python/lsst/daf/butler/formatters/jsonFormatter.py

```diff
@@ -51,7 +51,7 @@ def _readFile(self, path, pytype=None):
         """
         try:
             with open(path, "r") as fd:
-                data = json.load(fd)
+                data = self._fromBytes(fd.read())
```

Comment:
If `_fromBytes` is using bytes then you should have opened this file with `rb` mode.